services/secrets_manager.py

```diff
@@ -1,23 +1,72 @@
 from models import RuleCheckResult
 import boto3
+import datetime
+from dateutil.tz import tzlocal
 
 
-# client = boto3.client("")
+client = boto3.client("secretsmanager")
 
 
 def secretsmanager_rotation_enabled_check():
+    compliant_resources = []
+    non_compliant_resources = []
+    secrets = client.list_secrets()["SecretList"]
+
+    for secret in secrets:
+        if secret["RotationEnabled"] == True:
```

Comment:
`RotationEnabled` attribute가 존재하지 않을 수도 있습니다.
```suggestion
        if secret.get("RotationEnabled") == True:
```